README.md

```diff
@@ -214,8 +214,8 @@ Example:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214267597-8abe443a-7178-4011-854e-ae235fd91e25.png)

- [ ] You didn't add a link to your repository link with the clone command in this section in the image above :arrow_up: to make your readme more descriptive :+1: Those commands are given below :arrow_down:

```
git clone git@github.com:r-ahmed2022/bookstore.git
cd bookstore
```

Reply:
Done as advised